src/skills-builder/skills-builder-modal/view-results/CarouselStack.jsx

```diff
@@ -21,7 +21,7 @@ const CarouselStack = ({ selectedRecommendations }) => {
       const splitStrings = productType.split('_');
 
       // map through the array and normalize each string (i.e. ['Boot', 'Camp'])
-      const normalizeStrings = splitStrings.map(word => word[0].toUpperCase() + word.slice(1).toLowerCase());
+      const normalizeStrings = splitStrings.map(word => word[0].toUpperCase() + word.slice(1));
```

Comment:
Previously, "2U_degree" would appear as "2u Degree" in the UI. This makes it so it will read 2U Degree in the UI without effecting the other product types (lines of business)